actors/runtime/src/util/downcast.rs

```diff
@@ -85,15 +84,6 @@ fn downcast_util(error: anyhow::Error) -> anyhow::Result<ActorError> {
         Err(other) => other,
     };
 
-    // Check also for Cbor error to be safe. All should be converted to EncodingError, but to
-    // future proof.
-    let error = match error.downcast::<CborError>() {
```

Comment:
We no longer return this error anywhere in shared. Confirmed by grepping both the fvm crates and this crate for imports of serde_ipld_serde and removing all re-exports.